docs/pages/dashboard/_meta.json

```diff
@@ -1,4 +1,7 @@
 {
   "index": "Introduction",
```

Comment:
Let's not forget to add a section "dashboard" in the introduction page

Reply:
Agree, currently missing ;)